apps/docs/pages/guides/getting-started/tutorials/with-sveltekit.mdx

```diff
@@ -267,12 +267,23 @@ In the following code snippet, we perform the following steps:
 ```js title=src/routes/auth/callback/+server.js
 // src/routes/auth/callback/+server.js
 import { redirect } from '@sveltejs/kit'
+import { isAuthApiError } from '@supabase/supabase-js'
 
 export const GET = async ({ url, locals: { supabase } }) => {
   const code = url.searchParams.get('code')
 
   if (code) {
-    await supabase.auth.exchangeCodeForSession(code)
+    try {
+      await supabase.auth.exchangeCodeForSession(code)
+    } catch (error) {
+       // If opened in another browser without PCKE cookie, need to redirect user to login.
+       if (isAuthApiError(error)) {
```

Comment:
@kangmingtay, is this the only possible reason `exchangeCodeForSession` can throw an auth API error? also the email wouldn't be verified because there's no code verifier sent in the token exchange right?

Reply:
From my testing: the email is verified, but the uncaught auth error breaks the UI flow. 

Would be great to target the exact error case here if possible!

Reply:
@charislam for additional context, the official auth guide is redirecting on error as well: https://supabase.com/docs/guides/auth/server-side/email-based-auth-with-pkce-flow-for-ssr?framework=sveltekit

It's less specific, not checking that it's an auth error.